ionoscloud/resource_ipfailover.go

```diff
@@ -53,28 +61,31 @@ func resourceLanIPFailover() *schema.Resource {
 
 func resourceLanIPFailoverCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(services.SdkBundle).CloudApiClient
-	dcid := d.Get("datacenter_id").(string)
-	lanid := d.Get("lan_id").(string)
-	if lanid == "" {
-		diags := diag.FromErr(fmt.Errorf("'lan_id' is missing, please provide a valid lan ID "))
-		return diags
-	}
+	dcId := d.Get("datacenter_id").(string)
+	lanId := d.Get("lan_id").(string)
 	ip := d.Get("ip").(string)
 	nicUuid := d.Get("nicuuid").(string)
 
-	properties := &ionoscloud.LanProperties{}
+	// First, retrieve the existent IP Failover groups
+	lan, apiResponse, err := lanSvc.FindLanById(*client, ctx, dcId, lanId)
+	if err != nil {
+		return diag.FromErr(err)
+	}
 
-	properties.IpFailover = &[]ionoscloud.IPFailover{
-		{
-			Ip:      &ip,
-			NicUuid: &nicUuid,
-		}}
+	// Add the new IP failover group to the list
+	*lan.Properties.IpFailover = append(*lan.Properties.IpFailover, ionoscloud.IPFailover{
```

Comment:
seems odd to mutate the reponse pointer, can't we just work on properties.IpFailower form below and append to that?

Reply:
@cristiGuranIonos now we are working directly on the list that we retrieved: check 7a35821c28be88119348db39cb88c96db7c113b9